client/src/App.js

```diff
@@ -1,19 +1,38 @@
-import { useState } from 'react';
-import styled from 'styled-components';
+import { useState, useEffect } from 'react';
+import { v4 as uuid4 } from 'uuid';
 
 import Add from './pages/Add';
+import Home from './pages/Home';
+import { loadFromLocal, saveToLocal } from './lib/localStorage';
 
 function App() {
-  const [members, setMembers] = useState([]);
+  const [members, setMembers] = useState(loadFromLocal('members') ?? []);
+  const orderedMembers = members.slice().sort(compareFirstName);
+
+  useEffect(() => {
+    saveToLocal('members', orderedMembers);
+  }, [orderedMembers]);
 
   function addMember(member) {
-    setMembers([...members, { member }]);
+    const newMember = { ...member, id: uuid4() };
+    setMembers([...members, newMember]);
+  }
+
+  function compareFirstName(a, b) {
+    if (a.firstName === b.firstName) {
+      return 0;
+    } else if (a.firstName < b.firstName) {
+      return -1;
+    } else {
+      return 1;
+    }
   }
 
   return (
     <div>
```

Comment:
You don't really need the div here I guess. The `main` element is wrapping around the other components :)